cli/cdsctl/environment_group.go

```diff
@@ -0,0 +1,80 @@
+package main
+
+import (
+	"fmt"
+	"reflect"
+	"regexp"
+	"strings"
+
+	"github.com/spf13/cobra"
+
+	"github.com/ovh/cds/cli"
+	"github.com/ovh/cds/sdk/exportentities"
+)
+
+var (
+	environmentGroupCmd = cli.Command{
+		Name:  "group",
+		Short: "Manage CDS group linked to an environment",
+	}
+
+	environmentGroup = cli.NewCommand(environmentGroupCmd, nil,
+		[]*cobra.Command{
+			cli.NewCommand(environmentGroupImportCmd, environmentGroupImportRun, nil),
+		})
+)
+
+var environmentGroupImportCmd = cli.Command{
+	Name:  "import",
+	Short: "Import group linked to a CDS environment",
```

Comment:
Perhaps more details about what is "import" should be more friendly for user?  "delete all and import all."

Reply:
Not always delete all and import all as explained here https://github.com/ovh/cds/pull/1410/files/2ee7cbf013ebaf915fe99726cb01441e7ca59065#diff-c87bd071ac37a75b0b4d7ac50f2947d6R38

But yes, I agree, we can see together to find better way to explain import to user

Reply:
ok, I have not seen the force flag